test/test_helper.rb

```diff
@@ -4,7 +4,12 @@
 require "rgeo/proj4"
 require "common/factory_tests"
 require "psych"
-require "pry-byebug"
+
+begin
```

Comment:
are you sure you want to add that ?

Reply:
I'm not sure. I was thinking about removing it entirely because I don't use it and the reason for this is because it doesn't work on newer versions of Ruby. I should probably just remove the gem.